<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Modif</class>
 <widget class="QDialog" name="Modif">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-size: 24px;
background: #333;
}
QPushButton, QComboBox {
border-radius: 15px;
background: red;
color: white;
}


QPushButton:hover
{
background: black;
}




QToolButton {
border-radius: 30px;
background: red;
}

QLabel {
color: white;
}

QLineEdit 
{
background: transparent;
border: none;
color: #717072;
border-bottom: 1px solid #717072;
}</string>
  </property>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mati√®re :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Etat :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>c++</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>javascript</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sql</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>network</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Note :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_note">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
